wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss

```diff
@@ -0,0 +1,147 @@
+@use '../abstracts' as *;
+
+.home {
+  &__about {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    color: $color-white-100;
+    background-color: $color-blue-000;
+    @include full-width-background($color-blue-000);
+
+    @include breakpoint('large') {
+      flex-wrap: nowrap;
+      gap: rem(32px);
+    }
+
+    &::before,
+    &::after {
+      @include z-index('postBG');
+    }
+
+    &-content {
+      width: 100%;
+    }
+
+    .home__subtitle {
+      --home-subtitle-color: #{$color-white-100};
+      padding-top: rem(40);
+    }
+
+    &-description {
+      @extend %text-style-paragraph-large;
+      padding-bottom: rem(24);
+      color: $color-white-190;
+    }
+
+    &-people {
+      display: flex;
+      flex: auto;
+      flex-direction: row;
+      flex-wrap: wrap;
+    }
+
+    &-person {
+      // This is where you would style the individual items in the definition list
+      display: grid;
+      grid-template-areas: 
+        'image title'
+        'image bio';
+      grid-template-columns: 100px auto;
+      grid-template-rows: min-content auto;
+      column-gap: rem(16);
+      row-gap: rem(4);
+
+      @include breakpoint('medium') {
+        width: 50%;
+      }
+
+      @include breakpoint('large') {
+        width: 100%;
+      }
+
+      @include breakpoint('xlarge') {
+        width: 50%;
+      }
+
+
+      &:last-of-type {
+        padding-bottom: rem(40);
+
+        @include breakpoint('medium') {
+          padding-left: rem(20);
+        }
+
+        @include breakpoint('large') {
+          padding-left: 0;
+        }
+
+        @include breakpoint('xlarge') {
+          padding-left: rem(20);
+        }
+      }
+
+      &:first-of-type {
+        padding-bottom: rem(24);
+      }
+      
+      &-image {
+        grid-area: image;
+
+        /* stylelint-disable-next-line */
+        img {
+          width: rem(100);
+          height: rem(130);
+          object-fit: cover;
+        }
+      }
+
+      &-title {
+        grid-area: title;
+        color: $color-white-100;
+        @extend %text-style-heading-medium;
+      }
+
+      &-bio {
+        grid-area: bio;
+        color: $color-white-165;
+        @extend %text-style-label-small;
+      }
+    }
+
+    &-familyphoto {
+      // NOT WORKING
+      @include csis-block-full-width-wrapper;
+      // margin-left: calc(0 - (var(--container-padding)));
+      padding-bottom: rem(8);
+      
+      img {
+        width: calc(100% + var(--container-padding));
+        max-width: unset;
+      }
+
+      figcaption {
+        padding-top: rem(8);
+        padding-bottom: rem(72);
+        padding-left: var(--container-padding);
+        color: $color-white-165;
+        @extend %text-style-paragraph-medium-short;
+
+        @include breakpoint('large') {
+          padding-left: 0;
+        }
+      }
+
+      @include breakpoint('large') {
```

Comment:
If these are meant to apply to the familyphoto class, move them so they're after the base styles for `&-familyphoto` and before the `img` styles.